src/main/resources/metadata/listMetadata.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+   Copyright (c) 2022, WSO2 LLc. (http://www.wso2.org) All Rights Reserved.
+
+   WSO2 LLC. licenses this file to you under the Apache License,
+   Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+<template xmlns="http://ws.apache.org/ns/synapse" name="listMetadata">
+    <parameter defaultValue="" isMandatory="false" name="containerName"/>
```

Comment:
Aren't these two parameters mandatory?

Reply:
Hi Sanoj, you are right. Fixed.
Thanks!